pgd_core/templates/profile.html

```diff
@@ -5,6 +5,31 @@
 
 {% block content %}
 	<head> Welcome, {{full_name}} </head>
+
+	<div style="position:fixed; top:70px; right:90px; width:200px;">
+		<form method="post">{% csrf_token %}
+
+			{{ form.query.label }} <br>
+			{{ form.query}} <br>
+			{% for choice in form.search_type %}
+	  			{{ choice.choice_label }}
+	  			<span class="radio"> {{ choice.tag }} </span> <br>
+			{% endfor %}
+
+		</form>
+	</div> 
+
+<style>
```

Comment:
Move it back.  At this point, you should only make changes that have
functional benefits.  We'll take another PR or two for style issues later!

On Tue, Aug 18, 2015 at 3:43 AM, Ramana Subramanyam <
notifications@github.com> wrote:

> In pgd_core/templates/profile.html
> <https://github.com/osuosl/pgd/pull/42#discussion_r37285188>:
>
> > @@ -5,6 +5,31 @@
> >
> >  {% block content %}
> >  	<head> Welcome, {{full_name}} </head>
> > +
> > +	<div style="position:fixed; top:70px; right:90px; width:200px;">
> > +		<form method="post">{% csrf_token %}
> > +
> > +			{{ form.query.label }} <br>
> > +			{{ form.query}} <br>
> > +			{% for choice in form.search_type %}
> > +	  			{{ choice.choice_label }}
> > +	  			<span class="radio"> {{ choice.tag }} </span> <br>
> > +			{% endfor %}
> > +
> > +		</form>
> > +	</div>
> > +
> > +<style>
>
> Moved above because i thought of having the CSS before the table started ?
> Not sure why i've moved it there though. Should I move it back to the end
> or shall I leave it here ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/osuosl/pgd/pull/42/files#r37285188>.
>